templates/search/blocks/search_sort_and_view_options.html

```diff
@@ -39,8 +39,7 @@ <h2 class="search-sort-view__heading"><span class="sr-only">Display as:</span></
                     <span class="sr-only">Displaying results as a grid</span>
                 </span>
             {% else %}
-                <a href="?{% query_string_include 'display' 'grid' %}" id="gridView" search-type="{% with request.resolver_match.url_name as url_name  %} {% if url_name == 'search-featured' %} {{searchtabs.ALL.value}} {% elif url_name == 'search-catalogue' %}{{searchtabs.CATALOGUE.value}}{% elif url_name == 'search-website' %}
-{{searchtabs.WEBSITE.value}} {% endif %} {%endwith%}"  search-bucket="{{buckets.current.label}}" search-filter-name="Sort results" search-filter-value="Display results as a grid" class="search-sort-view__icon">
+                <a href="?{% query_string_include 'display' 'grid' %}" id="gridView" search-type="{%with request.resolver_match.url_name as url_name%}{% if url_name == 'search-featured' %}{{searchtabs.ALL.value}}{% elif url_name == 'search-catalogue' %}{{searchtabs.CATALOGUE.value}}{% elif url_name == 'search-website' %}{{searchtabs.WEBSITE.value}}{% endif %}{%endwith%}"  search-bucket="{{buckets.current.label}}" search-filter-name="Sort results" search-filter-value="Display results as a grid" class="search-sort-view__icon">
```

Comment:
```suggestion
 <a href="?{% query_string_include 'display' 'grid' %}" id="gridView" search-type="{% with request.resolver_match.url_name as url_name %}{% if url_name == 'search-featured' %}{{searchtabs.ALL.value }}{% elif url_name == 'search-catalogue' %}{{ searchtabs.CATALOGUE.value }}{% elif url_name == 'search-website' %}{{ searchtabs.WEBSITE.value }}{% endif %}{% endwith %}"  search-bucket="{{ buckets.current.label }}" search-filter-name="{{ form.sort_by.name }}" search-filter-value="Display results as a grid" class="search-sort-view__icon">
```
1) keep a space inside the template tag for readability `{{ <value> }} {% <value> %}`
2) `id="gridView"` casing not matching with `id ="listview"` and vice-versa. Please update casing to meet consistency and conventions. 
3) search-filter-name=`"{{ form.sort_by.name }}"` which is the field name instead of `"Sort results"` for consistency see using the field name in  example   [search_filter_trackking.js](https://github.com/nationalarchives/ds-wagtail/blob/develop/scripts/src/modules/analytics/search/search_filters_tracking.js) Thoughts ?

Reply:
I have applied these changes